pkgs/tools/misc/eza/default.nix

```
@@ -43,6 +46,8 @@ rustPlatform.buildRustPackage rec {
       --bash completions/bash/eza \
       --fish completions/fish/eza.fish \
       --zsh completions/zsh/_eza
+  '' + lib.optionalString exaAlias ''
+    ln -s $out/bin/eza $out/bin/exa
```

To be simpler:

```suggestion
    ln -s eza $out/bin/exa
```

It'll be rewritten to be relative anyway.

> `eza> rewriting symlink /nix/store/pdy2gxifbz4xp55wvf3y6a03g2bv6kwf-eza-0.11.0/bin/exa to be relative to /nix/store/pdy2gxifbz4xp55wvf3y6a03g2bv6kwf-eza-0.11.0`